content/zh/devops/_index.md

```diff
@@ -5,47 +5,47 @@ layout: "scenario"
 css: "scss/scenario.scss"
 
 section1:
-  title: KubeSphere DevOps offers end-to-end workflow and integrates popular CI/CD tools to boost delivery.
```

Comment:
Change the English source text from "KubeSphere DevOps offers end-to-end workflow" to "KubeSphere DevOps offers end-to-end workflows"